docker-compose.yml

```
@@ -12,6 +12,8 @@ services:
         - action: sync
           path: ./veterans
           target: /usr/src/app
+    volumes:
+     - ./veterans/coverage:/usr/src/app/coverage
```

yamllint

[warning] wrong indentation: expected 6 but found 5

(indentation)